t/rest_server/spec.yaml

```diff
@@ -14,11 +14,43 @@
 
 apiVersion: 'cli.abcxyz.dev/v1alpha1'
 kind: 'Template'
-
-desc:
-  'An example template for a simple HTTP/JSON REST server.'
+desc: 'An example template for a simple HTTP/JSON REST server.'
+inputs:
+  - name: 'automation_service_account'
+    desc: 'Automation Service Account (ex: [account_name]@[project_name].iam.gserviceaccount.com)'
```

Comment:
Should this be "project_id" rather than "project_name"?

Reply:
Changed to project_id.